api/src/server.js

```diff
@@ -18,6 +18,7 @@ app.use(corsMiddleware);
 app.options('*', corsMiddleware);
 
 app.use(bodyParser.json({ limit: '5mb' }));
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
@ht2 do you foresee any issues with this?

Reply:
@ht2 for your reference
https://github.com/expressjs/body-parser#bodyparserurlencodedoptions

> Returns middleware that only parses urlencoded bodies and **only looks at requests where the Content-Type header matches the type option.**

> Options
> ...
> **type**
> ... Defaults to `application/x-www-form-urlencoded`

Reply:
Noted - no can't see any big issues here

Reply:
Ok great thanks for checking 👍